src/pages/elements/User.js

```diff
@@ -2,7 +2,32 @@ import React from 'react';
 import Definition from '../../components/Definition';
 import Roles from './Roles';
 
+
+
 function User({ user }) {
+  var key;
+  if(user.public_gpg_key == null){
+    key = "-";
+  }
+  else if (user.public_gpg_key.length>= 50) {
+     key = <button onClick={downloadKey}> {user.public_gpg_key.substring(0,50) + '...'} </button>;
```

Comment:
i like this a lot!
it makes so much sense that there will be a download button.
but i found it less intuitive then i thought.

maybe we can have something more similar to `Show more...` and `Show less...` and a download button?
or something like that...
i'll leave that for you to decide :)

Reply:
I wanted to do this, the way I would normally do this in react would be with a state change, but this is a function not a class so it does not have states (as far as I know). The files are structured differently then I've seen before which is what I wanted to ask @jmelis about. I can look more into it and see if I can figure out how to do state changes with how these files are set up.